frontend_lib/src/component/PopinFixed/PopinFixedRightPart.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import classnames from 'classnames'
+import PropTypes from 'prop-types'
+
+class PopinFixedRightPart extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      currentChildren: props.menuItemList[0].id
```

Comment:
this isn't very safe. If props.menuItemList.length === 0, this code will fail.
I understand that PopinFixedRightPart should not be called with an empty props menuItemList but, in that case, you should specify it using propTypes.

Check what I did for frontend_lib/src/PopinFixed/PopinFixed.jsx
As usual, feel free to ask me for more info